bgp-monitor-action-pack/bgp-monitor/script.py

```diff
@@ -9,6 +9,8 @@
 from cloudvision.Connector.grpc_client import create_query
 from cloudvision.Connector.codec import Path, Wildcard
 
+CHECK_PEERS_CMDS = ['enable', 'show ip bgp summary vrf all', 'show bgp evpn summary']
```

Comment:
I ran the linter against this and there are a number of failures that need to be addressed. This can be done running `make lint` in the root of the repo.

```
./bgp-monitor-action-pack/bgp-monitor/script.py:15:1: E302 expected 2 blank lines, found 1
./bgp-monitor-action-pack/bgp-monitor/script.py:70:101: E501 line too long (124 > 100 characters)
./bgp-monitor-action-pack/bgp-monitor/script.py:73:11: E111 indentation is not a multiple of four
./bgp-monitor-action-pack/bgp-monitor/script.py:74:15: E271 multiple spaces after keyword
./bgp-monitor-action-pack/bgp-monitor/script.py:74:101: E501 line too long (103 > 100 characters)
./bgp-monitor-action-pack/bgp-monitor/script.py:76:53: W291 trailing whitespace
./bgp-monitor-action-pack/bgp-monitor/script.py:77:1: W293 blank line contains whitespace
./bgp-monitor-action-pack/bgp-monitor/script.py:79:11: E111 indentation is not a multiple of four
./bgp-monitor-action-pack/bgp-monitor/script.py:80:15: E271 multiple spaces after keyword
./bgp-monitor-action-pack/bgp-monitor/script.py:80:101: E501 line too long (110 > 100 characters)
./bgp-monitor-action-pack/bgp-monitor/script.py:81:15: E111 indentation is not a multiple of four
./bgp-monitor-action-pack/bgp-monitor/script.py:82:15: E111 indentation is not a multiple of four
./bgp-monitor-action-pack/bgp-monitor/script.py:82:51: W291 trailing whitespace
./bgp-monitor-action-pack/bgp-monitor/script.py:83:1: W293 blank line contains whitespace
./bgp-monitor-action-pack/bgp-monitor/script.py:85:11: E111 indentation is not a multiple of four
./bgp-monitor-action-pack/bgp-monitor/script.py:126:56: W292 no newline at end of file
```
These will cause failures in the internal pipeline if not addressed

Reply:
My bad, I didn't have Flake8 running in vs-code. I think I cleared them up. Please let me know if you find anything else.

Reply:
I think you're working off of the 2023.1/2 branch so you don't have the latest linting directives used in trunk. The line length limit was changed recently in trunk to max 100 from 120 so there are a couple of tfailures running kit against the trunk linting
```
./bgp-monitor-action-pack/bgp-monitor/script.py:72:101: E501 line too long (109 > 100 characters)
./bgp-monitor-action-pack/bgp-monitor/script.py:76:101: E501 line too long (106 > 100 characters)
./bgp-monitor-action-pack/bgp-monitor/script.py:77:101: E501 line too long (102 > 100 characters)
./bgp-monitor-action-pack/bgp-monitor/script.py:82:101: E501 line too long (113 > 100 characters)
./bgp-monitor-action-pack/bgp-monitor/script.py:83:101: E501 line too long (102 > 100 characters)
```